src/mintpy/prep_nisar.py

```diff
@@ -195,8 +204,8 @@ def common_raster_bound(input_files, utm_bbox=None):
         x_bounds.append([west, east])
         y_bounds.append([south, north])
     if not utm_bbox is None:
-        x_bounds.append([utm_bbox[0], utm_bbox[2]])
-        y_bounds.append([utm_bbox[1], utm_bbox[3]])
+        x_bounds.append([utm_bbox[1], utm_bbox[3]])
+        y_bounds.append([utm_bbox[0], utm_bbox[2]])
```

Comment:
I would suggest to modify `bbox_to_utm()` output to be in the order of (x0, y0, x1, y1) instead of modifying here, to be consistent with other bbox instances in the code.